docs/content/en/blog/news/january-2022-update.md

```diff
@@ -0,0 +1,130 @@
+---
+date: 2022-01-05
+title: "January 2022 update"
+linkTitle: "January 2022 update"
+weight: 997
+description: "Development status update to recap what happened in January"
+author: Le Van Nghia ([@nghialv](https://twitter.com/nghialv2607))
+---
+
+_Published by the PipeCD dev team every month, this update will provide you with news and updates about the project! Please click [here](/blog/2021/11/01/november-2021-update/) if you want to see the last status update._
+
+#### Happy New Year
+---
+First of all, PipeCD team would like to wish you all a very Happy New Year. 🥳
+
+2021 has been a great year for both PipeCD project and PipeCD team. Many features have been introduced by more than 2000 pull requests from 33 contributors. We would like to thank all those wonderful contributors.
+
+![](/images/january-2022-contributor-list.png)
+
+Stepping into 2022, PipeCD team looks forward to contributing even more to make PipeCD project better and more useful for many users in our OSS community.
+
+#### What's changed
+---
+
+Since the last report, PipeCD team has introduced 2 releases ([v0.22.0](https://github.com/pipe-cd/pipe/releases/tag/v0.22.0), [v0.23.0](https://github.com/pipe-cd/pipe/releases/tag/v0.23.0)). These releases bring many updates to help PipeCD becomes more stable as well as introduce some interesting features. In this blog post, we will recap those new features. For all other changes, please check out each release note to see.
+
+##### Deployment chain
```

Comment:
I'm aware that previous posts were using h4 and h5 for these. But honestly it's a bit unreadable. Especially h5, at a glance, it's the same as the content. What do you find on changing all heading levels?

```suggestion
### What's changed
---

Since the last report, PipeCD team has introduced 2 releases ([v0.22.0](https://github.com/pipe-cd/pipe/releases/tag/v0.22.0), [v0.23.0](https://github.com/pipe-cd/pipe/releases/tag/v0.23.0)). These releases bring many updates to help PipeCD becomes more stable as well as introduce some interesting features. In this blog post, we will recap those new features. For all other changes, please check out each release note to see.

#### Deployment chain
```

Reply:
### Before
![image](https://user-images.githubusercontent.com/19730728/148499863-d540a5dd-6f52-4bbe-992c-49343b5ae692.png)

### After
![image](https://user-images.githubusercontent.com/19730728/148499805-1ed49f84-3326-4de8-9626-beb02ed1c872.png)

Reply:
Nice idea. Let me adopt it!

Reply:
Updated both of them. Thank you.